.github/workflows/rust.yml

```diff
@@ -17,88 +17,57 @@ jobs:
           - --no-default-features
 
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v4
 
-      - uses: actions-rs/toolchain@v1
+      - uses: dtolnay/rust-toolchain@stable
         with:
-          profile: minimal
           toolchain: ${{ matrix.rust }}
-          override: true
 
-      - uses: actions-rs/cargo@v1
-        with:
-          command: build
-          args: -p hecs --all-targets ${{ matrix.features }}
+      - run: rustup install --profile minimal ${{ matrix.rust }} && rustup default ${{ matrix.rust }}
```

Comment:
Why not do this using the parameters of `dtolnay/rust-toolchain`?

Reply:
I started by calling `rustup` directly and ended up using `dtolnay/rust-toolchain`, this is a leftover that I forgot